hyperspy/tests/model/test_components2D.py

```diff
@@ -28,3 +28,136 @@ def test_values(self):
         assert_allclose(gt.max(), 0.0795774715459)
         assert_allclose(gt.argmax(axis=0)[0], 500)
         assert_allclose(gt.argmax(axis=1)[0], 500)
+
+
+class TestExpression2D:
+
+    def test_with_rotation(self):
+        g = hs.model.components2D.Expression(
+            "exp(-((x-x0)**2 / (2 * sx ** 2) + (y-y0)**2 / (2 * sy ** 2)))",
```

Comment:
How about moving the expression string to the class / make it global / fixture? Then there would be no need to check each expression that it's identical, and if there is something explicitly non-referenced, it's also clear.

May also be worth it making sure in at least one of the tests that setting the initial parameter values when initializing the component works.